lib/components/Tiles/Tiles.screenshots.tsx

```diff
@@ -34,9 +34,9 @@ export const screenshots: ComponentScreenshot = {
       Example: () => (
         <Tiles space={['none', 'small']} columns={[1, 2]} dividers>
           {[...new Array(2 * exampleRows)].map((_, i) => (
-            <Card key={i}>
+            <Box background="card" padding="gutter" key={i}>
```

Comment:
Would Placeholder be better here then?

Reply:
Because of the border on Placeholder it clashes with the divider. So felt the visual test was stronger like this